test/parallel/test-tls-client-mindhsize.js

```diff
@@ -54,7 +54,7 @@ function test(size, err, next) {
       client.on('error', function(e) {
         nerror++;
         assert.strictEqual(e.message,
-                           'DH parameter size 1024 is less than 2048');
+                           'ERR_DH_PARAM_SIZE');
```

Comment:
In this file, `e` is of type Object Error. This doesn't have the field `e.code` which we can access, so I believe, this one will be more appropriate in that case. 

I have written a small test-case and verified that:
```javascript
var u = require('util')
var e = new Error('ERR_DH_PARAM_SIZE');
console.log(u.inspect(e, {showHidden: true}))`
```
Output:
```{ Error: ERR_DH_PARAM_SIZE
 at Object.<anonymous> (/home/bidisha/swe_node/n1/node/foo.js:2:9)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3
  [stack]: 'Error: ERR_DH_PARAM_SIZE\n    at Object.<anonymous> (/home/bidisha/swe_node/n1/node/foo.js:2:9)\n    at Module._compile (module.js:569:30)\n    at Object.Module._extensions..js (module.js:580:10)\n    at Module.load (module.js:503:32)\n    at tryModuleLoad (module.js:466:12)\n    at Function.Module._load (module.js:458:3)\n    at Function.Module.runMain (module.js:605:10)\n    at startup (bootstrap_node.js:158:16)\n    at bootstrap_node.js:575:3',
  [message]: 'ERR_DH_PARAM_SIZE' }`
```

Please suggest on this.